src/icons/folder-icon/FolderIcon.js

```diff
@@ -0,0 +1,37 @@
+Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
this should have been converted to a js.flow file instead of trying to add the defineProperty